Point_set_3/include/CGAL/Point_set_3.h

```diff
@@ -330,8 +330,8 @@ class Point_set_3
   void clear()
   {
     m_base.clear();
-    boost::tie (m_indices, boost::tuples::ignore) = this->add_property_map<Index>("index", typename Index::size_type(-1));
-    boost::tie (m_points, boost::tuples::ignore) = this->add_property_map<Point>("point", Point (0., 0., 0.));
```

Comment:
Another way to fix that sort of conversion warning: use `Point(0.f, 0.f, 0.f)`. `0.f` and `0.` are the same value, and a float converts to a double without any loss of precision.

But I like your option (`ORIGIN`) best.